Daylight.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme

```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
any particular reason this scheme got deleted?

Reply:
Yeah, I don't think we should use Test schemes, testing from the main scheme makes more sense to me. 👍